build.gradle.kts

```diff
@@ -72,10 +72,8 @@ dependencies {
     testImplementation("org.mockito.kotlin:mockito-kotlin:${Versions.mockitoKotlin}")
 }
 
-java {
-    toolchain {
-        languageVersion.set(JavaLanguageVersion.of(JavaVersion.VERSION_17.majorVersion))
-    }
+kotlin {
+    jvmToolchain(17)
```

Comment:
No functional changes. Kotling Gradle Plugin exposes now a dedicated api to configure toolchains so I thought we could switch to it